docs/quickstart.md

```diff
@@ -0,0 +1,174 @@
+
+5. In your web browser, enter the IP address of your server and log in to RAGFlow.
+
+   > - With default settings, you only need to enter `http://IP_OF_YOUR_MACHINE` (**sans** port number) as the default HTTP serving port `80` can be omitted when using the default configurations.
+
+## Configure LLMs
+
+RAGFlow is a RAG solution, and it needs to work with an LLM to offer grounded, hallucination-free question-answering capabilities. For now, RAGFlow supports the following LLMs, and the list is expanding:
+
+- OpenAI
+- Tongyi-Qianwen
+- Moonshot
+- DeepSeek-V2
+
+>  RAGFlow also supports deploying LLMs locally using Ollama or Xinference, but this part is not covered in this quick start guide. 
+
+To add and configure an LLM: 
+
+1. Click on your logo on the top right of the page **>** **Model Providers**:
+
+   ![2 add llm](https://github.com/infiniflow/ragflow/assets/93570324/10635088-028b-4b3d-add9-5c5a6e626814)
+
+   > Each RAGFlow account is able to use **text-embedding-v2** for free, a small embedding model of Tongyi-Qianwen. This is why you can see Tongyi-Qianwen in the **Added models** list. And you may need to update your Tongyi-Qianwen API key at a later point.
+
+2. Click on the desired LLM and update the API key accordingly (DeepSeek-V2 in this case):
+
+   ![update api key](https://github.com/infiniflow/ragflow/assets/93570324/4e5e13ef-a98d-42e6-bcb1-0c6045fc1666)
+
+   *Your added models appear as follows:* 
+
+   ![added available models](https://github.com/infiniflow/ragflow/assets/93570324/d08b80e4-f921-480a-b41d-11832489c916)
+
+3. Click **System Mode Settings** to globally set the following models: 
+
+   - Chat model, 
+   - Embedding model, 
+   - Image-to-text model. 
+
+   ![system model settings](https://github.com/infiniflow/ragflow/assets/93570324/cdcc1da5-4494-44cd-ad5b-1222ed6acc3f)
+
+> Some of the small models, such as the image-to-text model **qwen-vl-max**, are subsidiary to a particular LLM. And you may need to update your API key accordingly to use these models. 
+
+## Create your first knowledge base
+
+You are allowed to upload files to a knowledge base in RAGFlow and parse them into datasets. A knowledge base is virtually a collection of datasets. Question answering in RAGFlow can be based on a particular knowledge base or multiple knowledge bases. For now, RAGFlow supports Word, slides, excel, txt, images, scanned copies, structured data, and web pages. 
+
+To create your first knowledge base:
+
+1. Click the **Knowledge Base** tab in the top middle of the page **>** **Create knowledge base**.
+
+2. Input the name of your knowledge base and click **OK** to confirm.
+
+   _You are taken to the **Configuration** page of your knowledge base._
+
+   ![knowledge base configuration](https://github.com/infiniflow/ragflow/assets/93570324/384c671a-8b9c-468c-b1c9-1401128a9b65)
+
+3. Select the embedding model and chunk (parsing) method for your knowledge base, and click **Save** to confirm your change. 
```

Comment:
Once embedding model is selected, it can't be changed any more.

Chunk(parsing) method need a explaination.